javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -505,6 +506,22 @@ public static ExecutableElement constructor(NewClassTree tree) {
         }
     }
 
+    /**
+     * Determines the type for a constructor at its call site given an invocation via {@code new}.
+     *
+     * @param tree the constructor invocation0
+     * @return the {@link ExecutableType} corresponding to the constructor call (i.e., the given
+     *     {@code tree}) at its call site
+     */
+    public static ExecutableType constructorType(NewClassTree tree) {
```

Comment:
For a `NewClassTree` we have a `elementFromUse` and a `constructor` method that do slightly different things.
Should this method be `typeFromUse` or `constructorType`? Or do we need both?
Can you clarify in the documentation to which element this corresponds?